environment_setup/setup_software.sh

```diff
@@ -114,7 +114,12 @@ print_status_msg "Setting Up Virtual Python Environment"
 # delete tbotspython first
 sudo rm -rf /opt/tbotspython
 
-if ! sudo /usr/bin/python3.8 -m venv /opt/tbotspython ; then
+if ! curl -sS https://bootstrap.pypa.io/get-pip.py | sudo /usr/bin/python3.10 ; then
+    print_status_msg "Error: Installing pip failed"
+    exit 1
+fi
```

Comment:
Do we need this: on ubuntu 24 ARM emulation I get the following error:

`curl -sS https://bootstrap.pypa.io/get-pip.py | sudo /usr/bin/python3.10
[sudo] password for arun: 
Sorry, try again.
[sudo] password for arun: 
Collecting pip
  Using cached pip-24.2-py3-none-any.whl.metadata (3.6 kB)
Using cached pip-24.2-py3-none-any.whl (1.8 MB)
Installing collected packages: pip
  Attempting uninstall: pip
    Found existing installation: pip 24.0
error: uninstall-no-record-file`

Reply:
CI needs it for some reason. `ensurepip` doesn't work either